snakemake/subworkflow.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+from snakemake.logging import logger
+from snakemake.exceptions import WorkflowError
+
+
+def subworkflow(
+    path,
+    basedir,
+    input,
+    output,
+    params,
+    wildcards,
+    threads,
```

Comment:
are threads enforced in sub-workflows?

Reply:
They should be (I hope), because I set `cores=threads` further below.